RHDEVDOCS-6199 - Builds for OpenShift: Install and Config Documentation Obsolete (#84027)
openshift/openshift-docs: modules/ob-installing-builds-with-operator.adoc

.. Check that the *Installed Namespace* is set to `openshift-builds` by default.

.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.

.. Select an *Update Channel*:

** The *Update Channel* is set to *latest* by default. The latest channel enables installation of the most recent stable version of the {builds-operator}.
** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.1`, you can use the `builds-1.1` channel.

. Click *Install*.

.Verification

After installing the Builds for Red Hat OpenShift Operator, you must verify that the following resources are created in the `openshift-builds` namespace to ensure proper functioning of the controllers:

* `OpenshiftBuild`

> * `ShipwrightBuild` 

Reviewer: nit: technically these are cluster-scoped resources, and don't belong to any namespace. I wouldn't block merge on this detail, as I don't think a cluster admin can really tell the difference in the OpenShift admin console.

. On the *Install Operator* page:

.. Check that the *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-builds` namespace to watch and be made available to all namespaces in the cluster.

.. Check that the *Installed Namespace* is set to `openshift-builds` by default.

.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.

.. Select an *Update Channel*:

** The *Update Channel* is set to *latest* by default. The latest channel enables installation of the most recent stable version of the {builds-operator}.
** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.1`, you can use the `builds-1.1` channel.

. Click *Install*.

.Verification

> After installing the Builds for Red Hat OpenShift Operator, you must verify that the following resources and pods are created to ensure proper functioning of the controllers:

Reviewer 2: Verification steps are instructional. Can you please check with SME/QE how and where in the console users can check these resources are created or not?

And;

Same comment around `Builds for Red Hat OpenShift Operator,` as earlier.

1. Capital B in `Build`
2. Usage of attribute for  Red Hat OpenShift Operator

> After installing the Builds for Red Hat OpenShift Operator, you must verify that the following resources and pods are created to ensure proper functioning of the controllers:

Reviewer 2: ```suggestion
After installing the Builds for Red Hat OpenShift Operator, you must verify that the following resources are created to ensure proper functioning of the controllers:
```

Same comment to remove pods because pod is a resource itself.

> Resources:

Reviewer 2: ```suggestion

```

This is redundant. It can be removed.

* `OpenshiftBuild`
* `ShipwrightBuild` 

> The following pods must be created in `openshift-builds` namespace:

Reviewer 2: How can user navigate to the `openshift-builds` namespace? Is it straightforward in order to skip it?

Author: The pods are installed in openshift-builds namespace. This is explained in the installation steps.

> The following pods must be created in `openshift-builds` namespace:

Reviewer 2: ```suggestion
Verify that the following pods are created in the `openshift-builds` namespace by.....
```
The verification section is missing numbered step. 
Consider adding steps and phrasing it like an instruction.

> * openshift-builds operator pod

Reviewer 2: ```suggestion
* The `openshift-builds` Operator pod
```

* Build controller and build webhook pods

> * SharedResource daemonset and SharedResource webhook pods

Reviewer 2: - Terms are missing back ticks. 

`openshift-builds` `SharedResource`-- should be in back ticks. But please confirm with the SME/QE.

- > Do not start a sentence with a lowercase term; rewrite the sentence.

To avoid start a sentence with a lowercase term, 

"The `openshift-builds` Operator pod"
